### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by GinoVin**
[DOC-12296](https://coveord.atlassian.net/browse/DOC-12296)

## Proposed changes

Some small updates to wording in the readme file 

## Breaking changes

Should not cause any breaking changes

## Testing

- [ ] Unit Tests:

- [ ] Functionnal Tests:

- [ ] Manual Tests:

[DOC-12296]: https://coveord.atlassian.net/browse/DOC-12296?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by louis-bompart** [comment]
Hey @GinoVin , please create a Jira and put it in the PR description :)

**Comment by GinoVin** [comment]
> Hey @GinoVin , please create a Jira and put it in the PR description :)

Done!

**Comment by GinoVin** [comment]
I don't think I'll be able to merge the changes here @louis-bompart you might have to complete the merge once you appprove the changes I suggested :)

File: packages/ui/atomic/tempdoc/README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
+    * To create new pages in the `src/pages` folder, ensure you import the JS and CSS (refer to the `src/pages/index.html` of your generated project).
 2.  `npm start`
 3.  ???
 4.  Profit.
 
 ## Publishing
 
 1.  Open a command prompt in the directory of the component.
-    > For example, if I created a component named `omelette-au-fromage`, I need to open a command prompt in `src/components/omelette-au-fromage`.
+    * Example: You created a component named `omelette-au-fromage`, you then need to open a command prompt in `src/components/omelette-au-fromage`.
```
Comment (btaillon) [inline packages/ui/atomic/tempdoc/README.md:33]: I'm a bit confused about the wording here.

```suggestion
    * Example: If you created a component named `omelette-au-fromage`, you would need to open a command prompt in `src/components/omelette-au-fromage`.
```

**Review by btaillon — APPROVED** [review]

**Pull request merged**